src/components/treePopup/index.tsx

```diff
@@ -149,31 +154,48 @@ const TreePopup: React.FC<TreePopupProps> = ({
               )}
               <GreyText>{treeInfo.address}</GreyText>
               {(() => {
-                switch (treeInfo.id) {
-                  case NO_TREE_PRESENT:
+                let editSiteButton = <GreenLinkButton
+                                  to={`${ParameterizedRouteBases.SITE}${treeInfo.id}`}
+                                  state={{ destination: Routes.MY_TREES }}
+                                  target="_blank"
+                                >
+                                  Edit Site Page
+                                </GreenLinkButton>;
+
+                if (!treeInfo.treePresent) {
+                  
                     return (
-                      <PlantRequest>
-                        Want to plant a tree here?{' '}
-                        <Typography.Link
-                          href={CITY_PLANTING_REQUEST_LINK}
-                          target="_blank"
-                        >
-                          Submit a request to the city!
-                        </Typography.Link>
-                      </PlantRequest>
+                      <div>
+                        <PlantRequest>
+                          Want to plant a tree here?{' '}
+                          <Typography.Link
+                            href={CITY_PLANTING_REQUEST_LINK}
+                            target="_blank"
+                          >
+                            Submit a request to the city!
+                          </Typography.Link>
+                        </PlantRequest>
+                        {userIsAdmin && editSiteButton}
+                      </div>
                     );
-                  default:
+                }
+                const StyledDiv = styled.span`
+                  marginLeft: .1rem;
+                `;
```

Comment:
⛏️ Can you declare this outside of the component? Traditionally keep these at the top of the file.